Functions.Templates/Templates/EventGridTrigger-Python/function.json

```diff
@@ -0,0 +1,10 @@
+{
+  "bindings": [
+    {
+      "type": "eventGridTrigger",
+      "name": "event",
+      "direction": "in"
+    }
+  ],
+  "disabled": false

```

Comment:
Remove "disabled":false as the V2 runtime does not respect it anymore.

Reply:
It is present in all other templates though.  Should it be removed everywhere for consistency?

Reply:
Yes, it needs to be removed everywhere else as well. I'll create a PR for that separately.